README.md

```diff
@@ -407,7 +407,13 @@ docker pull downloads.unstructured.io/unstructured-io/unstructured-api:latest
 Once pulled, you can launch the container as a web app on localhost:8000.
 
 ```bash
-docker run -p 8000:8000 -d --rm --name unstructured-api downloads.unstructured.io/unstructured-io/unstructured-api:latest --port 8000 --host 0.0.0.0
+docker run -p 8000:8000 -d --rm --name unstructured-api downloads.unstructured.io/unstructured-io/unstructured-api:latest
+```
+
+You can pass in a PORT variable to run the server on a different port in the container. Don't forget to expose this new port via Docker.
```

Comment:
Nitpick: 
> Don't forget to expose this new port via Docker.

This wording makes me think I'd have to do some additional work, beyond what is already in example in line 416 (`-e PORT=9500`). Some docker beginners could think they need to add EXPOSE in Dockerfile :smile:   I think the first sentence is enough.